docs/images.md

```diff
@@ -0,0 +1,25 @@
+# Images and their usage in MLRun
+
+Every release of MLRun includes several images for different usages. The build and the infrastructure images are described, and located, in the [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.md). They are also published to [dockerhub](https://hub.docker.com/u/mlrun) and [quay.io](https://quay.io/organization/mlrun).
+
+## Using images
+
+See [Kubernetes Jobs & Images](./runtimes/mlrun_jobs.ipynb)
+
+## MLRun images and how to build them 
+
+See [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.md).
+
+## MLRun images and external docker images
+
+There is no difference in the usage between the MLRun images and external docker images. However, MLRun images:
+- Resolve auto tags: If you specify ```image="mlrun/mlrun"``` the API fills in the tag by its version, e.g. changes it to `mlrun/mlrun:0.9.2`. And if the API gets upgraded you'll automatically get a new API image. 
```

Comment:
We're about to change the behavior so that the tag will be filled by the API, **but with the version of the client**, so I guess worth changing it here and only merge it after the change is done

Reply:
when is the change?

Reply:
Part of it already happened in https://github.com/mlrun/mlrun/pull/1707
The other part should be in by Sunday EOD